### ISSUE TITLE
there is No way to download the modified sound that we can hear on speakers - that audio cannot be downloaded

### ISSUE THREAD

**Issue by albertkeys** [TuckerBurke/WebAudioMixer#2]
there is No way to download the modified sound that we can hear on speakers - that audio cannot be downloaded

**Comment by TuckerBurke** [comment]
Correct. This is a live mixing platform, not a DAW, much like turntables with a mixer don't record natively. Hypothetically, you could record your mix using any existing DAW (such as ProTools or Ableton) by routing your machine's audio into their i/o. Glad someone stumbled upon this piece of software!

**Comment by albertkeys** [comment]
Hai thanks. But i would like to have that feature. I can pay if you wish.  Let me know.
I need it for my own purpose.

Thanks in advance

**Comment by albertkeys** [comment]
Hai i have emailed you as well. I look forward to you helping in adding that download button
Looking forward to your response

Thanks in advance
